src/js/src/components/SearchSingleItemComponents/SearchSingleItemAllData.vue

```diff
@@ -5,51 +5,121 @@
         {{ allDataButtonText }}
       </button>
     </div>
-    <div v-if="allDataShown">
+    <div v-if="allDataShown && allData !== {}">
       <hr class="informationDivider">
       <div class="table">
         <div class="tr">
           <span class="td highlightText">Attribute</span><span class="td highlightText">Value</span>
         </div>
-        <div v-for="(key, index) in Object.entries(result)" :key="index" class="tr">
-          <span class="td">{{ key[0] }}</span> <span class="td">{{ key[1] }}</span>
+        <div v-for="(key, index) in Object.entries(allData)"
+             :key="index"
+             class="tr">
+          <span class="td">{{ key[0] }}</span>
+          <div v-if="key[1].constructor === Array"
```

Comment:
Is it important that the constructor is Array or is it just enough to know if any data is present in the array/object?

Reply:
I've used that same method of checking if it's an array or not a little above this. Essentially, Im handling arrays one way, and strings/objects/numbers an entirely other way. This was a good way to check whether i was dealing with an array or not. I could probably have done the check in a couple of other ways too, but i liked this one.

Reply:
If you want to know if it is array or not this is fine. I was uncertain about wether this is always an array - in that case other checks would be better.